things/api.py

```diff
@@ -10,7 +10,7 @@
 import os
 from shlex import quote
 from typing import Dict, List, Union
-
+import urllib.parse
```

Comment:
add newline afterwards; the imports are split into three sections: stdlibrary, third-party, and finally local.

WIth `pip install isort` installed you can run `make auto-style` (see the Makefile).

Reply:
Will fix